CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - Add validation for subjects in RBAC RoleBinding and ClusterRoleBinding.
 - Fixed a bug where single-letter subscriptions were not allowed, even though
 they were intended to be.
+- Fixed an issue when a GraphQL query could fail if a namespace contained a
+large number of events.
```

Comment:
Sort of an incorrect and lackadaisical entry on my part. It would be more correct for me to say 'Fixed an issue where a GraphQL query could fail when querying a namespace containing event data in excess of 2GBs. '